Registry/Registry.EM_COMMON

```diff
@@ -2338,6 +2353,7 @@ rconfig   integer urban_map_bd            derived         	1             1
 rconfig   integer urban_map_wd            derived         	1             1        rh    "urban_map_wd"    "urban mapping 7: ind_wd"       ""  
 rconfig   integer urban_map_gbd           derived         	1             1        rh    "urban_map_gbd"   "urban mapping 8: ind_gbd"      ""  
 rconfig   integer urban_map_fbd           derived         	1             1        rh    "urban_map_fbd"   "urban mapping 9: ind_fbd"      ""  
+rconfig   integer urban_map_zgrd          derived               1             1        rh    "urban_map_zgrd"  "urban mapping 10: ind_grd"      ""
```

Comment:
the variables urban_map_{zrd, zwd, gd, zd, zdf, bd, wd, gbd, fdb} are all defined in the NMM Registry. To be conventional, `urban_map_zgrd` should also be there.